mentoring-wg/gsoc-selection-and-ranking-process.md

```diff
@@ -0,0 +1,28 @@
+# Contributor+proposal selection and ranking process
+
+The selection and ranking process for GSoC Contributors and proposals is performed by the CNCF GSoC administrators, who will use information provided by mentors to make the final decision. The baseline of this process is outlined by Google in their [slot allocation](https://developers.google.com/open-source/gsoc/help/slot-allocation) and [organization admin tips](https://developers.google.com/open-source/gsoc/help/oa-tips#reviewing_gsoc_contributor_proposals) documents.
+
+To summarize:
+- Mentors mark the applicants they would like to mentor. They do that based on the quality of the proposals, their belief in the ability of the applicant successfully doing the project and also their experience from the interaction with the applicant so far.
+- CNCF GSoC admins select and rank the proposals and submit their requests to Google (detailed below)
+- Google allocates a number of slots to CNCF and fills the slots based on the rankings provided by CNCF GSoC admins.
+
+The selection and ranking process of CNCF GSoC admin consists of three passes:
+
+- First pass: Ignore the proposals that had no mentor interest
+- Second pass: Review the proposals and identify the ones that are not a good fit for the program. Google's [rule](https://developers.google.com/open-source/gsoc/help/slot-allocation#important_notes_about_slot_requests) is to only accept proposals that are good or excellent, rather than just "okay".
+- Third pass: Rank the proposals based on the following criteria (not in any particular order):
+    - Existing community interaction of the applicant
+    - Number of mentors who want to mentor the applicant
+    - Proposal quality (lower weight, as GSoC admins might not know the technical details of the project)
+    - Impact and alignment of the project idea
+    - Number of proposals for a community (to avoid overloading a community)
```

Comment:
We should add something around the applicant profile, as in @tanayvaswani 's feedback

Reply:
Is there a profile as such to review? I know a candidate's name is provided, but other than that it's only the proposal that is shared in the system (IIRC).

Reply:
The profile is often shared in the proposal. Who the applicant is, what kind of contributions they've made to the project already, or to other projects. I will just add a single sentence about this in the criteria.